README.md

```diff
@@ -21,48 +21,52 @@ The project adheres to [Semantic Versioning](https://semver.org) and [Convention
 
 ## Table of Content
 
-- [Design Principles](#design-principles)
```

Comment:
It seems the formatting is wrong. Also I am using tab of 2 for indentation. I think I need to state that in the contribution doc. :)